src/Adapter/Category/CommandHandler/UpdateCategoryPositionHandler.php

```diff
@@ -67,7 +67,7 @@ public function handle(UpdateCategoryPositionCommand $command)
             throw new CategoryNotFoundException($command->getCategoryId(), sprintf('Category with id "%s" was not found', $categoryId));
         }
 
-        if ($category->updatePosition($command->getWay(), $position)) {
+        if ($category->updatePosition((bool) $command->getWay(), $position)) {
```

Comment:
```
/**
     * @var int
     */
    private $way;
```

🤔

Reply:
No parameter for category.

But `updatePosition` waits for a boolean parameter
![image](https://user-images.githubusercontent.com/1533248/103867204-4027db80-50c7-11eb-9001-75e33a058c96.png)